src/browser-resolution/resolving-domains-in-a-browser.md

```diff
@@ -15,52 +15,52 @@ Here are some of the end user scenarios that should give an idea which features
 
 ### Distributed Website Browsing
 
-1. Given a blockchain domain has an dweb content identifier record configured \(e.g. IPFS hash\).
-2. When user enters the domain name into a browser address bar.
+1. Given a blockchain domain has a dweb content identifier record configured \(e.g. IPFS hash\).
+2. The user enters the domain name into a browser address bar.
 3. Then the browser resolves the domain and gets the content hash of a domain.
 4. Then the browser retrieves the content by the hash using a related protocol and displays the content.
```

Comment:
Bogdan used such style deliberately, to conform with BDD's "Given-When-Then"

Reply:
Ok we can change that back, but in I don't like the wording & repetitiveness with these lists. Having the first 3 steps of every list be 90% similar makes me want to skip what I'm reading.

I'd suggest we get rid of steps 1 & 2 everywhere on this page and use a format like this: 
```md
### HTTP Website Browsing

**Given:** A blockchain domain has a DNS record configured.
**When:** The user enters the domain name into a browser address bar.
1. The browser resolves the domain and gets the specified DNS records.
2. Then the browser requests and displays the content using DNS protocol and HTTP protocol.
```

Also "When the user enters the domain name into a browser address bar." Is step 2 for the 1st 3 lists, it seems like it'd be more appropriate to group all 3 of these lists some way and have that step be a given for all of them.

Reply:
Looking at more of the "Given-When-Then" I didn't see any of them use lists. Most of them were structured like Psedo code:
```
**Scenario**: lorem ipsum
Given:
When:
Then:
And/Then:
```